script.js

```diff
@@ -0,0 +1,22 @@
+document.getElementById('check-btn').addEventListener('click', () => {
+  const userInput = document.getElementById('user-input').value;
+  const resultsDiv = document.getElementById('results-div');
+
+  if (userInput === '') {
+    resultsDiv.innerText('PleaSe provide a phone number!');
+    return;
+  }
+
+  const regex = /^(1\s?)?(\(\d{3}\)|\d{3})[-\s]?\d{3}[-\s]?\d{4}$/;
+
+  if (regex.test(userInput)) {
+    resultsDiv.innerText = `Valid US number: ${userInput}`;
+  } else {
+    resultsDiv.innerText = `Invalid US number: ${userInput}`;
+  }
+});
```

Comment:
- In here for empty input your code is not working instead you have to write your code like this:

```javascript
document.getElementById('check-btn').addEventListener('click', () => {
  const userInput = document.getElementById('user-input').value;
  const resultsDiv = document.getElementById('results-div');

  if (userInput === '') {
    resultsDiv.innerText = 'PleaSe provide a phone number!';
  } else {
    const regex = /^(1\s?)?(\(\d{3}\)|\d{3})[-\s]?\d{3}[-\s]?\d{4}$/;

    if (regex.test(userInput)) {
      resultsDiv.innerText = `Valid US number: ${userInput}`;
    } else {
      resultsDiv.innerText = `Invalid US number: ${userInput}`;
    }
  }
});
```
- In here I update the innerHtml to use = sign not ().
- And also I used else for your conditional statement.